docs/Parallel-Learning-Guide.rst

```diff
@@ -451,6 +451,17 @@ Example
 
 -  `A simple distributed learning example`_
 
+Ray
+^^^
+
+`Ray`_ is a Python-based framework for distributed computing. The `lightgbm_ray`_ project, maintained within the official Ray GitHub organization, can be used to perform distributed LightGBM training using ``ray``.
+
+See `the lightgbm_ray documentation`_ for more usage examples.
```

Comment:
Is it correct to use `more` here? We haven't provided any examples, but say "for more".
```suggestion
See `the lightgbm_ray documentation`_ for usage examples.
```

Reply:
Yes because the link `lightgbm_ray` in the line above links to https://github.com/ray-project/lightgbm_ray, which contains example code in the README.

![image](https://user-images.githubusercontent.com/7608904/132082956-92b8fb91-34d4-4e7f-b250-66d97be33dc5.png)

Reply:
Oh, this is not so obvious, to be honest. And example code can be moved out from the `lightgbm_ray`'s README at any time in the future. But OK, I understood what `more` actually means here, thanks.